packages/compass-e2e-tests/package.json

```diff
@@ -68,9 +68,9 @@
     "nyc": "^15.1.0",
     "prettier": "^2.7.1",
     "ps-list": "^8.1.0",
-    "puppeteer-core": "^23.10.3",
+    "puppeteer-core": "^22.15.0",
```

Comment:
So webdriverio specifies 22 causing ELSPROBLEMS, but seems like we really need 23 here, otherwise firefox fails to install itself, hmmm